pom.xml

```diff
@@ -50,7 +50,7 @@
     <site>
```

Comment:
I tried commenting this out when I was testing, but it caused mvn site to error, so I put it back in.